Jenkinsfile

```diff
@@ -63,35 +55,39 @@ node('linux && docker') {
     }
 
     withDockerContainer(image: 'node:14', args: '-u=root') {
-      stage('Commit bumped version') {
-        withCredentials([
-        usernameColonPassword(credentialsId: 'github-commit-token', variable: 'GH_CREDENTIALS')]) {
-          sh 'npm run bump:version:pre'
+      if (!isBump) {
+        stage('Commit bumped version') {
+            withCredentials([
+            usernameColonPassword(credentialsId: 'github-commit-token', variable: 'GH_CREDENTIALS')]) {
+              sh 'npm run bump:version:pre'
+            }
         }
-      }
-
-      stage('Npm publish') {
-        withCredentials([
-        string(credentialsId: 'NPM_TOKEN', variable: 'NPM_TOKEN')]) {
-          sh "echo //registry.npmjs.org/:_authToken=${NPM_TOKEN} > ~/.npmrc"
-          sh 'npm run npm:publish:alpha || true'
+      } else {
+        stage('Npm publish') {
+          withCredentials([
+          string(credentialsId: 'NPM_TOKEN', variable: 'NPM_TOKEN')]) {
+            sh "echo //registry.npmjs.org/:_authToken=${NPM_TOKEN} > ~/.npmrc"
+            sh 'npm run npm:publish:alpha || true'
+          }
         }
       }
     }
 
     withDockerContainer(image: '458176070654.dkr.ecr.us-east-1.amazonaws.com/jenkins/deployment_package:v7') {
-      stage('Veracode package') {
-        sh 'rm -rf veracode && mkdir veracode'
+      if (isBump) {
```

Comment:
Wouldn't it be simpler to return/end the pipeline after the `bump` stage is complete? There wouldn't be a need for the `else` or for this `if`.